tests/ui_tests/test_numbers.py

```diff
@@ -20,7 +20,7 @@
 
 
 @parametrize_by_data_subdirs(__file__, 'bundles')
-def test_number_validation(sdk_client_fs: ADCMClient, path, app_fs, login_to_adcm):
+def test_number_validation(sdk_client_fs: ADCMClient, path, app_fs, api_auth_to_adcm):
```

Comment:
Replace other occurrences of `login_to_adcm` fixture.
Also rename fixture `login_to_adcm` to `login_to_adcm_over_ui` for the further strict separation of `login_to_adcm_over_ui` and new fixture

Reply:
done